.travis.yml

```diff
@@ -0,0 +1,42 @@
+addons:
+  apt:
+    packages:
+      - expect-dev
+
+language: python
+
+python:
+  - "2.7"
+
+virtualenv:
+  system_site_packages: true
+
+env:
+  global:
+    - PROJECT="zpl2"
+    - VERSION="1.0.0"
+    - RELEASE="1.0.0"
+    - AUTHOR_NAME="Sylvain Garancher"
+    - PYPI_USER="syleam"
+    # Generate the below per repo with the following:
+    #   travis encrypt 'PYPI_PASSWORD="$PYPI_PASSWORD"' -r LasLabs/repo-name
+    # - secure: "$PYPI_TOKEN"
+    # Generate the below per repo with the following:
+    #   travis encrypt 'GH_TOKEN="$GITHUB_TOKEN"' -r LasLabs/repo-name
```

Comment:
If you want Github docs, just do this and reply with the secure token so I can add:

```
travis encrypt 'GH_TOKEN="GITHUB TOKEN"' -r syleam/python-zpl2
```

The Github token would be made in Github account settings, and will need write access to this repo.

You'll also need to make a `gh-pages` branch & clear it out like so - https://help.github.com/articles/creating-project-pages-using-the-command-line/

The bot will handle the rest, as you can [see here](https://github.com/laslabs/Python-CFSSL/tree/gh-pages). I recommend using a bot user instead of your own.